### PR TITLE
fix(openai-compat): ignore usage:null in stream token accounting

### PR DESCRIPTION
**Pull request by LuZhuanz**
## Summary
- Fix OpenAI-compatible stream usage parsing to ignore `usage:null` chunks.
- Add stream parser fallbacks for wrapped usage paths and response-style token fields.
- Add regression tests for chat/responses usage and `usage:null` case.

## Root Cause
Upstream streaming responses often include many chunks with `usage:null` before the final chunk that contains real usage.
The previous parser treated `usage:null` as valid usage and published zero tokens early.
Because usage publishing is once-only, later real usage was ignored, so management usage stayed `0`.

## Changes
- Only parse usage when the usage node is a JSON object.
- Keep compatibility for both token field styles:
  - `prompt_tokens` / `completion_tokens`
  - `input_tokens` / `output_tokens`
- Add fallback parsing for wrapped paths:
  - `usage`
  - `response.usage`
  - `data.usage`
  - `result.usage`
  - `message.usage`

## Tests
- Added parser tests for:
  - OpenAI chat-completions style stream usage
  - OpenAI responses style stream usage
  - `usage:null` stream chunk ignored behavior

## Validation
- `go test ./internal/runtime/executor/helps -run 'ParseOpenAI(Stream)?Usage|NullUsage' -v`
- `go build -o test-output ./cmd/server && rm test-output`
- Local end-to-end reproduction (dashscope OpenAI-compatible):
  - final stream chunk contains non-zero usage
  - management usage changes from `0` to non-zero after fix

### CODE REVIEW

File: internal/runtime/executor/helps/usage_helpers.go [hunk 1]
```diff
@@ -242,17 +242,45 @@ func ParseOpenAIStreamUsage(line []byte) (usage.Detail, bool) {
 	}
 	usageNode := gjson.GetBytes(payload, "usage")
 	if !usageNode.Exists() {
+		usageNode = gjson.GetBytes(payload, "response.usage")
+	}
+	if !usageNode.Exists() {
```
[the remark this answers is not in the document]
Comment (LuZhuanz) [reply internal/runtime/executor/helps/usage_helpers.go:247]: @copilot apply changes based on [this feedback](https://github.com/router-for-me/CLIProxyAPI/pull/2760#discussion_r3071642949)
